<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneratorWsad√≥w</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AppIcon.qrc">
    <normaloff>:/icons/applications-system.png</normaloff>:/icons/applications-system.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="InputPathBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="SaveButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="PatternComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="PointsLabel">
      <property name="text">
       <string>Points type:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="BeepButton">
      <property name="cursor">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
      <property name="text">
       <string>Beep</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="RevievEditBox"/>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="DialogButton">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Add new point/pattern</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="OpenButton">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="KergEdit">
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="OutputPathBox">
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="PointsComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="PatternLabel">
      <property name="text">
       <string>Patterns:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="KergCheckBox">
      <property name="text">
       <string>Kerg:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AppIcon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OpenButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onButtonOpenClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onButtonSaveClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onButtonDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonOpenClick()</slot>
  <slot>onButtonSaveClick()</slot>
  <slot>onButtonDialog()</slot>
 </slots>
</ui>
